### ISSUE TITLE
cri-o integration tests failing on openSUSE

### ISSUE THREAD

**Issue by marcov** [kata-containers/tests#1940]
Large part of `ctr.bats` tests are failing on openSUSE.

The pkg/cgroups package recently integrated in cri-o expects an unified crgroup hierarchy (cgroupv2), but openSUSE is still an hybrid hierarchy (basically a mix of cgroup v1 and v2).

For more info: https://github.com/containers/libpod/issues/3912

This is impacting CI in runtime repo.

Maybe we can disabled running `ctr.bats` on openSUSE while a proper solution is  found?

/cc @chavafg @GabyCT

**Comment by GabyCT** [comment]
I disable opensuse with CRI-O to avoid more CI failures

**Comment by marcov** [comment]
I checked the logs again, and what I though was an error about cgroups was just a `warning`:
```
/tmp/tmp.6S3MFHZmfK/containers/exits --socket-dir-path /tmp/tmp.6S3MFHZmfK/containers --log-level debug --runtime-arg --root=/run/runc -t]"
# time="2019-09-02 15:19:04.938413700Z" level=warning msg="Failed to add conmon to cgroupfs sandbox cgroup: write /sys/fs/cgroup/unified/Burstable/pod_123-456/crio-conmon-a82e1e9cc523f04238cf81443ede06ffa4e79ca45ea958dc841fe4694e4b9370/tasks: open /sys/fs/cgroup/unified/Burstable/pod_123-456/crio-conmon-a82e1e9cc523f04238cf81443ede06ffa4e79ca45ea958dc841fe4694e4b9370/tasks: permission denied"
# time="2019-09-02 15:19:25.317099400Z" level=debug msg="Received container pid: -1"
# time="2019-09-02 15:19:25.317154200Z" level=error msg="Container creation error: rpc error: code = Unavailable desc = transport is closing\n"
# time="2019-09-02T15:19:25Z" level=fatal msg="Creating container failed: rpc error: code = Unknown desc = container create failed: rpc error: code = Unavailable desc = transport is closing\n"
```

So @GabyCT we should try to understand what's wrong here...

**Comment by chavafg** [comment]
this is the same issue we are seeing on Debian #1943

**Comment by chavafg** [comment]
this was fixed with #1953, closing this issue.
